tor/core/inbox.py

```diff
@@ -112,9 +113,17 @@ def check_inbox(config):
                 process_override(reply, config)
                 reply.mark_read()
                 return
+
             if 'good bot' in reply.body.lower() or 'bad bot' in reply.body.lower():
                 # please stop emailing me, I just don't care
                 reply.mark_read()
+                return
+
+            send_to_slack(
+                'Unknown reply by {author}: {body}'.format(
```

Comment:
The only request I have is to also add the subject of the message. Other than that it looks solid.